src/httpProblemDetailsDefinition.ts

```diff
@@ -0,0 +1,5 @@
+export interface HttpProblemDetailsDefinition {
+  type: string;
```

Comment:
According to RFC, this should be URI (including `about:blank`) - can we highlight it by the type here?

Reply:
What do you mean exactly, something like this:
```
type URI = string
```
?